src/urllib3/contrib/securetransport.py

```diff
@@ -500,31 +523,33 @@ def handshake(
                 result = Security.SSLHandshake(self.context)
 
                 if result == SecurityConst.errSSLWouldBlock:
-                    raise socket.timeout("handshake timed out")
+                    raise socket.timeout("handshake timed out")  # type: ignore
                 elif result == SecurityConst.errSSLServerAuthCompleted:
                     self._custom_validate(verify, trust_bundle)
                     continue
                 else:
                     _assert_no_error(result)
                     break
 
-    def fileno(self):
+    def fileno(self) -> int:
         return self.socket.fileno()
 
     # Copy-pasted from Python 3.5 source code
-    def _decref_socketios(self):
+    def _decref_socketios(self) -> None:
         if self._io_refs > 0:
             self._io_refs -= 1
         if self._closed:
             self.close()
 
-    def recv(self, bufsiz):
+    def recv(self, bufsiz: int) -> List[bytes]:
```

Comment:
This should be `-> bytes`?

Reply:
I think so but if I change it to `bytes` I get
`src/urllib3/contrib/securetransport.py:548: error: Incompatible return value type (got "List[bytes]", expected "bytes")`

Is it related to `data = buffer[:bytes_read]`?

Reply:
Probably? Might require a cast on the `return`?

Reply:
Done!